pages/glossary/bounding-box-around-text.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Bounding Box around Text
+key: bounding-box-around-text
+---
+
+A rectangle where the width is the [advance width](https://www.w3.org/TR/css-values/#length-advance-measure) of the characters, plus one pixel on the left and right, and the height is the [advance height](https://www.w3.org/TR/css-values/#length-advance-measure) of the characters, plus one pixel on the top and bottom.
```

Comment:
I don't think that https://www.w3.org/TR/css-values/#length-advance-measure defines advance width or advance height. It just states that the advance measure is the advance width or height, but doesn't say anything about advance width or height. The note below defines what advanced height is. A similar one for advanced width is needed.

Reply:
I think the advanced measures definition properly defines what the advanced with and height are. We can make that clearer if you'd like. But with respect to the note, I don't know of a way to get the advanced with using CSS. There are ways to do it with `canvas`, but I that just means computing the with in pixels of the "0" glyph, like the advanced measures definition says. My proposal would be to leave this as is.